packages/ilp-protocol-stream/test/connection.spec.ts

```diff
@@ -267,14 +262,12 @@ describe('Connection', function () {
       stream2.on('end', clientSpy.stream2.end)
       stream2.on('close', clientSpy.stream2.close)
 
-      stream1.setSendMax(100)
```

Comment:
Why was this removed?

Reply:
By mistake I think - I added it back in.